kubetest/main.go

```diff
@@ -459,7 +487,7 @@ func prepare() error {
 		return err
 	}
 
-	if err := os.MkdirAll("./_artifacts", 0777); err != nil { // Create artifacts
+	if err := os.MkdirAll(artifacts, 0777); err != nil { // Create artifacts
```

Comment:
Yeah good catch! But that's only for unit tests. It isn't used by the e2e tests. 

Also I'm not changing the behavior here (it is _artifacts before and after this PR). The difference is that I'm no longer using a string literal.